projects/packages/search/src/instant-search/components/search-app.jsx

```diff
@@ -189,7 +186,10 @@ class SearchApp extends Component {
 	showResults = this.toggleResults.bind( this, true );
 
 	onChangeQueryString = isHistoryNav => {
-		this.getResults();
+		// `null` means the searchQuery is reset or not initialized.
+		if ( this.props.searchQuery !== null ) {
+			this.getResults();
+		}
```

Comment:
This appears to prevent the search overlay from fetching results when only filter values are defined (e.g. `/?category=classic`).

Reply:
@jsnmoon good catch. ah right... let me think..

Reply:
@jsnmoon I'll reverse the change and leave it for the params to be reviewed all tegether. Thanks

Reply:
Sounds good to me — let's do a holistic review of all the parameters in the future.